### PR TITLE
add velero integration doc

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add document about how to use velero to backup karmada resources.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by RainbowMango** [comment]
We need a document about how to backup and restore resources between clusters.

**Comment by gy95** [comment]
> We need a document about how to backup and restore resources between clusters.

fixed

File: docs/working-with-velero.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+cloud platform or on-premises.
+
+Velero lets you:
+
+- Take backups of your cluster and restore in case of loss.
+- Migrate cluster resources to other clusters.
+- Replicate your production cluster to development and testing clusters.
+
+This document gives an example to demonstrate how to use the `Velero` to back up and restore Kubernetes cluster resources
+and persistent volumes. I'll backup Kubernetes resources in `member1`, and then restore those to `member2`.
```
Comment (RainbowMango) [inline docs/working-with-velero.md:14]:
```suggestion
and persistent volumes. Following example backups resources in cluster `member1`, and then restore those to cluster `member2`.
```

File: docs/working-with-velero.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+And then run the below command to switch to the member cluster `member1`.
+```shell
+export KUBECONFIG=/root/.kube/members.config
+kubectl config use-context member1
+```
+
+## Install MinIO
+Velero uses Object Storage Services from different cloud providers to support backup and snapshot operations.
+For simplicity, I'll take `MinIO`, one object storage that runs locally on k8s clusters, for an example.
```
Comment (RainbowMango) [inline docs/working-with-velero.md:30]:
```suggestion
For simplicity, here takes `MinIO`, one object storage that runs locally on k8s clusters, for an example.
```

File: docs/working-with-velero.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+nginx   2/2     2            2           21s
+```
+
+For more details about Velero, you can search for in [Velero Github Repo](https://github.com/vmware-tanzu/velero)
+or [Velero Website](https://velero.io/)
+
+To finish this doc, I reference the official doc from [MinIO](https://github.com/minio/minio) 
+and [Velero](https://github.com/vmware-tanzu/velero), and some contents are copied directly from documents from two communities, such as
+[Velero Doc](https://velero.io/docs/v1.7/contributions/minio/), [MinIO Doc](https://github.com/minio/minio/blob/master/README.md#gnulinux).
+Thanks a lot for the support of both two communities.
```
Comment (RainbowMango) [inline docs/working-with-velero.md:166]:
```suggestion
## Reference

The above introductions about `Velero` and `MinIO` are only a summary from the official website and repos, for more details please refer to:
- Velero: https://velero.io/
- MinIO: https://min.io/

```

Comment (gy95) [reply docs/working-with-velero.md]: all done.

**Pull request merged**